sdks/go/pkg/beam/core/graph/fn.go

```diff
@@ -168,7 +168,9 @@ const (
 	restrictionSizeName          = "RestrictionSize"
 	createTrackerName            = "CreateTracker"
 
-	createWatermarkEstimatorName = "CreateWatermarkEstimator"
+	createWatermarkEstimatorName          = "CreateWatermarkEstimator"
+	getInitialWatermarkEstimatorStateName = "GetInitialWatermarkEstimatorState"
+	getWatermarkEstimatorStateName        = "GetWatermarkEstimatorState"
```

Comment:
Drop the "Get" prefixes. Idiomatic Go doesn't prefix field fetching methods with "Get" unnecessarily.

Reply:
Updated - I took the rest of your changes as well, thanks for the feedback!